livekit-agents/livekit/agents/tts/tts.py

```diff
@@ -82,7 +89,13 @@ async def aclose(self) -> None:
         self._event_ch.close()
 
     async def __anext__(self) -> SynthesizedAudio:
-        return await self._event_ch.__anext__()
+        if self._timeout > 0:
+            try:
+                return await asyncio.wait_for(self._event_ch.__anext__(), self._timeout)
+            except (TimeoutError, asyncio.TimeoutError) as e:
```

Comment:
```suggestion
            except asyncio.TimeoutError as e:
```
asyncio.TimeoutError will cover both (Python >= 3.9)